Description: mention onsgmls in the docs,
 everywhere that sgmls and nsgmls are mentioned now
Origin: vendor
Bug-Debian: https://bugs.debian.org/804941
Forwarded: no
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2015-11-13

--- a/SGMLS.pm
+++ b/SGMLS.pm
@@ -5,7 +5,7 @@
 
 =head1 NAME
 
-SGMLS - class for postprocessing the output from the B<sgmls> and
+SGMLS - class for postprocessing the output from the B<onsgmls>, B<sgmls>, and
 B<nsgmls> parsers.
 
 =head1 SYNOPSIS
@@ -102,7 +102,7 @@
 
 This class holds a single parse.  When you create an instance of it,
 you specify a file handle as an argument (if you are reading the
-output of B<sgmls> or B<nsgmls> from a pipe, the file handle will
+output of B<onsgmls>, B<sgmls> or B<nsgmls> from a pipe, the file handle will
 ordinarily be C<STDIN>):
 
   my $parse = new SGMLS(STDIN);
@@ -124,11 +126,11 @@
 
 =item C<file()>: Return a string containing the name of the current
 SGML source file (this will work only if the C<-l> option was given to
-B<sgmls> or B<nsgmls>).
+B<onsgmls>, B<sgmls> or B<nsgmls>).
 
 =item C<line()>: Return a string containing the current line number
 from the source file (this will work only if the C<-l> option was
-given to B<sgmls> or B<nsgmls>).
+given to B<onsgmls>, B<sgmls> or B<nsgmls>).
 
 =item C<appinfo()>: Return a string containing the C<APPINFO>
 parameter (if any) from the SGML declaration.
--- a/DOC/HTML/SGMLSpm/definition.html
+++ b/DOC/HTML/SGMLSpm/definition.html
@@ -10,7 +10,7 @@
 
 <P><A HREF=sgmlspm.html><TT>SGMLS.pm</TT></A> is an <A HREF=extend.html>extensible</A> <A HREF="http://www.metronet.com/0/perlinfo/perl5/manual/perl.html"><TT>perl5</TT></A>
 class library for parsing the output from James Clark's popular
-<TT>sgmls</TT> and <A HREF="http://www.jclark.com/sp.html"><TT>nsgmls</TT></A> parsers, available on the Internet at <A HREF="ftp://jclark.com/"><TT>ftp://jclark.com</TT></A>.
+<TT>onsgmls</TT>, <TT>sgmls</TT> and <A HREF="http://www.jclark.com/sp.html"><TT>nsgmls</TT></A> parsers, available on the Internet at <A HREF="ftp://jclark.com/"><TT>ftp://jclark.com</TT></A>.
 This is <EM>not</EM> a complete system for translating
 documents written the the <I>Standard Generalised Markup
 Language</I> (<A HREF="http://www.sil.org/sgml/sgml.html"><B>SGML</B></A>) into other formats, but it can easily
--- a/DOC/HTML/SGMLSpm/events.html
+++ b/DOC/HTML/SGMLSpm/events.html
@@ -146,7 +146,7 @@
 class (which will contain a list of attributes, etc. -- see
 below), <TT>$event->file</TT> and
 <TT>$event->line</TT> will return the file and line-number
-in which the element appeared (if you called <TT>sgmls</TT> or <A HREF="http://www.jclark.com/sp.html"><TT>nsgmls</TT></A> with
+in which the element appeared (if you called <TT>onsgmls</TT>, <TT>sgmls</TT> or <A HREF="http://www.jclark.com/sp.html"><TT>nsgmls</TT></A> with
 the <IT>-l</IT> flag), and
 <TT>$event->element</TT> will return the element currently
 in force (in this case, the same as
--- a/DOC/HTML/SGMLSpm/sgmls.html
+++ b/DOC/HTML/SGMLSpm/sgmls.html
@@ -22,7 +22,7 @@
 <PRE>use SGMLS;</PRE>
 </P>
 <P>You must then open up a file handle from which <A HREF=sgmlspm.html><TT>SGMLS.pm</TT></A> can read the
-data from an <TT>sgmls</TT> or <A HREF="http://www.jclark.com/sp.html"><TT>nsgmls</TT></A> process, unless you are reading from
+data from an <TT>onsgmls</TT>, <TT>sgmls</TT> or <A HREF="http://www.jclark.com/sp.html"><TT>nsgmls</TT></A> process, unless you are reading from
 a standard handle like <TT>STDIN</TT> -- for example,
 if you are piping the output from <TT>sgmls</TT> to a <A HREF="http://www.metronet.com/0/perlinfo/perl5/manual/perl.html"><TT>perl5</TT></A> script, using
 something like</P>
@@ -37,7 +37,7 @@
 <A HREF=sgmlspm.html><TT>SGMLS.pm</TT></A> doesn't need to seek, so any input stream should
 work.</P>
 
-<P>To parse the <TT>sgmls</TT> or <A HREF="http://www.jclark.com/sp.html"><TT>nsgmls</TT></A> output from the handle, create
+<P>To parse the <TT>onsgmls</TT>, <TT>sgmls</TT> or <A HREF="http://www.jclark.com/sp.html"><TT>nsgmls</TT></A> output from the handle, create
 a new object instance of the <TT>SGMLS</TT> class with
 the handle as an argument, i.e.</P>
 
--- a/DOC/HTML/SGMLSpm/sgmlsevent.html
+++ b/DOC/HTML/SGMLSpm/sgmlsevent.html
@@ -114,7 +114,7 @@
 
 
 <P>The <TT>file</TT> and <TT>line</TT> methods
-will return useful information only if you called <TT>sgmls</TT> or <A HREF="http://www.jclark.com/sp.html"><TT>nsgmls</TT></A>
+will return useful information only if you called <TT>onsgmls</TT>, <TT>sgmls</TT> or <A HREF="http://www.jclark.com/sp.html"><TT>nsgmls</TT></A>
 with the <IT>-l</IT> flag to include file and
 line-number information.</P>
 
--- a/DOC/HTML/sgmlspl/handlers.html
+++ b/DOC/HTML/sgmlspl/handlers.html
@@ -61,7 +61,7 @@
 </P>
 <P>You will not always need to use the <IT>event</IT>
 argument, but it can be useful if you want to report line numbers or
-file names for errors (presuming that you called <TT>sgmls</TT> or <A HREF="http://www.jclark.com/sp.html"><TT>nsgmls</TT></A>
+file names for errors (presuming that you called <TT>onsgmls</TT>, <TT>sgmls</TT> or <A HREF="http://www.jclark.com/sp.html"><TT>nsgmls</TT></A>
 with the <IT>-l</IT> option).  If you have a new version
 of <A HREF="http://www.jclark.com/sp.html"><TT>nsgmls</TT></A> which accepts the <IT>-h</IT> option, you
 can also use the <IT>event</IT> argument to look up
--- a/DOC/sgmlspl.sgml
+++ b/DOC/sgmlspl.sgml
@@ -11,6 +11,7 @@
   <!ENTITY output.pm "<link linkend='output'><application>SGMLS::Output</application></link>">
   <!ENTITY sgmlspl "<link linkend='sgmlspl'><application>sgmlspl</application></link>">
   <!ENTITY perl5 "<ulink url='http://www.metronet.com/0/perlinfo/perl5/manual/perl.html'><application>perl5</application></ulink>">
+  <!ENTITY onsgmls "<application>onsgmls</application>">
   <!ENTITY sgmls "<application>sgmls</application>">
   <!ENTITY nsgmls "<ulink url='http://www.jclark.com/sp.html'><application>nsgmls</application></ulink>">
   <!ENTITY LaTeX SDATA "[LaTeX]">
@@ -231,7 +232,7 @@
 
 <para>You will not always need to use the <parameter>event</parameter>
 argument, but it can be useful if you want to report line numbers or
-file names for errors (presuming that you called &sgmls; or &nsgmls;
+file names for errors (presuming that you called &onsgmls; or &sgmls; or &nsgmls;
 with the <parameter>-l</parameter> option).  If you have a new version
 of &nsgmls; which accepts the <parameter>-h</parameter> option, you
 can also use the <parameter>event</parameter> argument to look up
--- a/DOC/sgmlspm.sgml
+++ b/DOC/sgmlspm.sgml
@@ -12,6 +12,7 @@
   <!ENTITY sgmls.pm "<link linkend=SGMLSpm><application>SGMLS.pm</application></link>">
   <!ENTITY perl5 "<ulink url='http://www.metronet.com/0/perlinfo/perl5/manual/perl.html'><application>perl5</application></ulink>">
   <!ENTITY perl5 "<application>perl5</application>">
+  <!ENTITY onsgmls "<application>onsgmls</application>">
   <!ENTITY sgmls "<application>sgmls</application>">
   <!ENTITY nsgmls "<ulink url='http://www.jclark.com/sp.html'><application>nsgmls</application></ulink>">
 ]>
@@ -40,7 +41,7 @@
 </artheader>
 
 <para>Welcome to &sgmls.pm;, an extensible &perl5; class library for
-processing the output from the &sgmls; and &nsgmls; parsers.
+processing the output from the &onsgmls;, &sgmls;, and &nsgmls; parsers.
 &sgmls.pm; is free, copyrighted software available by anonymous ftp in
 the directory <ulink
 url="ftp://aix1.uottawa.ca/pub/dmeggins/">ftp://aix1.uottawa.ca/pub/dmeggins/</ulink>.
@@ -75,7 +76,7 @@
 
 <para>&sgmls.pm; is an <link linkend=extend>extensible</link> &perl5;
 class library for parsing the output from James Clark's popular
-&sgmls; and &nsgmls; parsers, available on the Internet at <ulink
+&onsgmls;, &sgmls;, and &nsgmls; parsers, available on the Internet at <ulink
 url="ftp://jclark.com/"><filename>ftp://jclark.com</filename></ulink>.
 This is <emphasis>not</emphasis> a complete system for translating
 documents written the the <glossterm>Standard Generalised Markup
@@ -100,7 +101,7 @@
 <title>How do I produce &sgml; documents?</title>
 
 <para>I am presuming here that you are already experienced with &sgml;
-and the &sgmls; or &nsgmls; parser.  For help with the parsers see the
+and the &onsgmls;, &sgmls; or &nsgmls; parser.  For help with the parsers see the
 manual pages accompanying each one; for help with &sgml; see Robin
 Cover's SGML Web Page at <ulink
 url="http://www.sil.org/sgml/sgml.html"><filename>http://www.sil.org/sgml/sgml.html</filename></ulink>
@@ -143,7 +144,7 @@
 </programlisting>
 
 <para>You must then open up a file handle from which &sgmls.pm; can read the
-data from an &sgmls; or &nsgmls; process, unless you are reading from
+data from an &onsgmls;, &sgmls; or &nsgmls; process, unless you are reading from
 a standard handle like <symbol>STDIN</symbol> &mdash; for example,
 if you are piping the output from &sgmls; to a &perl5; script, using
 something like</para>
@@ -159,7 +160,7 @@
 &sgmls.pm; doesn't need to seek, so any input stream should
 work.</para>
 
-<para>To parse the &sgmls; or &nsgmls; output from the handle, create
+<para>To parse the &onsgmls;, &sgmls; or &nsgmls; output from the handle, create
 a new object instance of the <classname>SGMLS</classname> class with
 the handle as an argument, i.e.</para>
 
@@ -265,7 +266,7 @@
 </table>
 
 <para>The <command>file</command> and <command>line</command> methods
-will return useful information only if you called &sgmls; or &nsgmls;
+will return useful information only if you called &onsgmls;, &sgmls; or &nsgmls;
 with the <parameter>-l</parameter> flag to include file and
 line-number information.</para>
 
@@ -372,7 +373,7 @@
 class (which will contain a list of attributes, etc. &mdash; see
 below), <literal>$event->file</literal> and
 <literal>$event->line</literal> will return the file and line-number
-in which the element appeared (if you called &sgmls; or &nsgmls; with
+in which the element appeared (if you called &onsgmls;, &sgmls; or &nsgmls; with
 the <parameter>-l</parameter> flag), and
 <literal>$event->element</literal> will return the element currently
 in force (in this case, the same as
--- a/sgmlspl.1
+++ b/sgmlspl.1
@@ -4,7 +4,7 @@
 .\"
 .\" ----------------------------------------------------------------------
 .SH "NAME"
-sgmlspl \- a simple post-processor for nsgmls
+sgmlspl \- a simple post-processor for onsgmls/nsgmls
 .\"
 .\" ----------------------------------------------------------------------
 .SH "SYNOPSIS"
@@ -24,7 +24,7 @@
 were doc.latex, then use the following command in a Unix shell to
 convert the SGML document:
 
-nsgmls doc.sgml | sgmlspl spec.pl > doc.latex
+onsgmls doc.sgml | sgmlspl spec.pl > doc.latex
 
 sgmlspl will pass any additional arguments on to the specification
 file, which can then process them in the regular perl5 fashion.
@@ -38,7 +38,7 @@
 .BR /usr/share/doc/sgmls-doc/SGMLSpm/sgmls.html
 SGMLS documentation in HTML format
 .TP
-.BR nsgmls (1)
+.BR onsgmls (1)
 .\"
 .\" ----------------------------------------------------------------------
 .SH "AUTHOR"
--- a/DOC/LaTeX/sgmlspm.tex
+++ b/DOC/LaTeX/sgmlspm.tex
@@ -17,7 +17,7 @@
 
 
 Welcome to {\sc SGMLS.pm}, an extensible {\sc perl5} class library for
-processing the output from the {\sc sgmls} and {\sc nsgmls} parsers.
+processing the output from the {\sc onsgmls}, {\sc sgmls}, and {\sc nsgmls} parsers.
 {\sc SGMLS.pm} is free, copyrighted software available by anonymous ftp in
 the directory ftp://aix1.uottawa.ca/pub/dmeggins/.
 You might also want to look at the documentation for {\sc sgmlspl},
@@ -55,7 +55,7 @@
 
 {\sc SGMLS.pm} is an extensible {\sc perl5}
 class library for parsing the output from James Clark's popular
-{\sc sgmls} and {\sc nsgmls} parsers, available on the Internet at {\tt ftp://jclark.com}.
+{\sc onsgmls}, {\sc sgmls}, {\sc nsgmls} parsers, available on the Internet at {\tt ftp://jclark.com}.
 This is {\em not\/} a complete system for translating
 documents written the the {\em Standard Generalised Markup
 Language\/} ({\sc SGML}) into other formats, but it can easily
@@ -78,7 +78,7 @@
 
 
 I am presuming here that you are already experienced with {\sc SGML}
-and the {\sc sgmls} or {\sc nsgmls} parser.  For help with the parsers see the
+and the {\sc onsgmls}, {\sc sgmls} or {\sc nsgmls} parser.  For help with the parsers see the
 manual pages accompanying each one; for help with {\sc SGML} see Robin
 Cover's SGML Web Page at {\tt http://www.sil.org/sgml/sgml.html}
 on the Internet.
@@ -121,7 +121,7 @@
 \end{verbatim}}
 
 You must then open up a file handle from which {\sc SGMLS.pm} can read the
-data from an {\sc sgmls} or {\sc nsgmls} process, unless you are reading from
+data from an {\sc onsgmls}, {\sc sgmls} or {\sc nsgmls} process, unless you are reading from
 a standard handle like {\tt STDIN} {---} for example,
 if you are piping the output from {\sc sgmls} to a {\sc perl5} script, using
 something like
@@ -137,7 +137,7 @@
 {\sc SGMLS.pm} doesn't need to seek, so any input stream should
 work.
 
-To parse the {\sc sgmls} or {\sc nsgmls} output from the handle, create
+To parse the {\sc onsgmls}, {\sc sgmls} or {\sc nsgmls} output from the handle, create
 a new object instance of the {\tt SGMLS} class with
 the handle as an argument, i.e.
 
@@ -192,7 +192,7 @@
 \end{tabular}\end{table}
 
 The {\tt file} and {\tt line} methods
-will return useful information only if you called {\sc sgmls} or {\sc nsgmls}
+will return useful information only if you called {\sc onsgmls}, {\sc sgmls} or {\sc nsgmls}
 with the {\tt\sl -l\/} flag to include file and
 line-number information.
 
@@ -235,7 +235,7 @@
 class (which will contain a list of attributes, etc. {---} see
 below), {\tt \$event->file} and
 {\tt \$event->line} will return the file and line-number
-in which the element appeared (if you called {\sc sgmls} or {\sc nsgmls} with
+in which the element appeared (if you called {\sc onsgmls}, {\sc sgmls} or {\sc nsgmls} with
 the {\tt\sl -l\/} flag), and
 {\tt \$event->element} will return the element currently
 in force (in this case, the same as
